logicmonitor_data_sdk/api/logs.py

```diff
@@ -165,9 +167,18 @@ def _do_request(self):
       self.UnLock()
 
   def _merge_request(self, single_request):
+    # size limiting
+    payload_cache = json.dumps(self._payload_cache)
+    compressed_payload_cache = gzip.compress(payload_cache.encode("utf-8"))
+    serialized_single_request = self.api_client.sanitize_for_serialization(single_request)
+    single_request_json = json.dumps(serialized_single_request)
```

Comment:
same as https://github.com/logicmonitor/logicmonitor_data_sdk_py/pull/19/files#r962836481

Reply:
Resolve at other places and files as well

Reply:
added try-except blocks for json.dumps() in all files.